### PR TITLE
Fixes for 0.11

### PR DESCRIPTION
**Pull request by tudorms**
- Disable Stats Timer (addresses #92)
- Include revision as part of product version in addition to suffix (addresses #91)
- Include license file in NuGet (addresses #98)
- Add Source Link to linker (addresses #102)

Note: I will cherry-pick this into the main branch as well, but we want to (at least temporarily) rev a RN 0.68-compatible 0.11 NuGet first.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/hermes-windows/pull/103)

### CODE REVIEW

**Comment by tudorms** [comment]
@microsoft/jshost can you take a look?

**Review by vmoroz — APPROVED** [review]
:shipit:

File: CMakeLists.txt [hunk 1]
```diff
@@ -459,6 +465,22 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
   endif()
 endif()
 
+if ("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
```
Comment (vmoroz) [inline CMakeLists.txt:468]: >"${CMAKE_C_COMPILER_ID}" [](http://example.com/codeflow?start=4&length=24)

Just for my education: why we do not use CMAKE_C_COMPILER_ID directly?

File: cmake/modules/SourceLink.cmake [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# over HTTP should work.
+#
+include(GitCommands)
+
+# Warn if this is included and the compilier doesn't support source link
+if ("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
+    if ("${CMAKE_C_COMPILER_VERSION}" VERSION_GREATER_EQUAL "19.20")
+        # Good to go!
+    elseif("${CMAKE_C_COMPILER_VERSION}" VERSION_GREATER_EQUAL "19.14")
+        message(STATUS "SourceLink enabled but case insensative")
```
Comment (vmoroz) [inline cmake/modules/SourceLink.cmake:26]: >insensative [](http://example.com/codeflow?start=52&length=11)

nit: spelling?

File: cmake/modules/SourceLink.cmake [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        return()
+    endif()
+
+    string(REPLACE ".git" "" RAW_GIT_URL ${GIT_REMOTE})
+    string(REPLACE "github.com" "raw.githubusercontent.com" RAW_GIT_URL ${RAW_GIT_URL})
+    string(CONCAT RAW_GIT_URL ${RAW_GIT_URL} "/${GIT_CURRENT_HASH}/*")
+
+    set(${OUTPUT} "\"${LOCAL_PATH}\" : \"${RAW_GIT_URL}\"" PARENT_SCOPE)
+
+endfunction(build_source_link_rule)
```
Comment (vmoroz) [inline cmake/modules/SourceLink.cmake:107]: >build_source_link_rule [](http://example.com/codeflow?start=12&length=22)

is it valid?

Comment (tudorms) [reply cmake/modules/SourceLink.cmake:107]: Yes, but it looks like the old style, I'll remove it: [https://cmake.org/cmake/help/latest/command/endfunction.html#command:endfunction](https://cmake.org/cmake/help/latest/command/endfunction.html#command:endfunction)

File: CMakeLists.txt (hunk 1 above)
Comment (tudorms) [reply CMakeLists.txt:468]: I think it's generally preferred for strings to be enclosed in quotes (especially if the string might contain a semicolon). But probably in this case it doesn't matter (will keep it for consistency with other uses in the file, like the if above).

[https://stackoverflow.com/questions/35847655/when-should-i-quote-cmake-variables](https://stackoverflow.com/questions/35847655/when-should-i-quote-cmake-variables)

**Comment by dannyvv** [comment]
Hey Tudor,

you might also have ot trigger the release pipeline as well as I think that has a branch filter for main/*-stable as well.

Thanks,
    Danny

**Pull request merged**